src/hooks/useExplorerData.ts

```diff
@@ -18,52 +33,50 @@ export function useExplorerData() {
 
   const { nullifier, subsetRoot } = debouncedRecentWithdrawal;
   const { accessType, bitLength, subsetData: data } = subsetMetadata;
+  
+  const [explorerData, setExplorerData] = useState<ExplorerDataI>(defaultExplorerData);
+  
+  useEffect(() => {
+    if (
+      !nullifier ||
+      !subsetRoot ||
+      !debouncedCommitments.length ||
+      !accessType
```

Comment:
If compared side by side, I removed the requirement that

```
isNaN(bitLength) ||
    data.length === 0
```

From the subset query, there seems to be valid subsets that don't return bitlengths or data.